chef/cookbooks/glance/templates/default/glance-api.conf.erb

```diff
@@ -81,6 +81,7 @@ workers = <%= [node["cpu"]["total"], 2, 4].sort[1] %>
 
 # Supported values for the 'container_format' image attribute
 #container_formats=ami,ari,aki,bare,ovf,ova
+container_formats=ami,ari,aki,bare,ovf,ova<%= @use_docker ? ",docker" : "" %>
```

Comment:
I think it should be nicer to have it as an attribute array and join it within the config, what do you think?